### ISSUE TITLE
ARM使能Landscape

### ISSUE THREAD

**Issue by ZhengZhenyu** [kunpengcompute/kunpengcompute.github.io#41]
ARM使能Landscape

大数据
--------

Field | Project | Status | Links
----- | -------- | ------ | ------
Big Data | Hadoop | ARM CI | https://ci-hadoop.apache.org/job/hadoop-qbt-linux-ARM-trunk/
Big Data | Hadoop | 软件包发布 | https://hadoop.apache.org/docs/r3.3.0/index.html
Big Data | Spark | ARM CI | https://amplab.cs.berkeley.edu/jenkins/label/spark-arm/
Big Data | Hive | ARM CI | https://ci-hadoop.apache.org/job/Hive-trunk-linux-ARM/
Big Data | HBase | ARM CI | https://ci-hadoop.apache.org/job/HBase/job/HBase-Nightly-ARM/
Big Data | Kudu | ARM CI | http://status.openlabtesting.org/builds?project=apache%2Fkudu
Database | MariaDB | ARM CI |  https://buildbot.mariadb.org/#/builders
AI | TensorFlow | ARM CI | https://github.com/tensorflow/tensorflow#community-supported-builds
AI | Pytorch | ARM CI |

**Comment by Yikun** [comment]
持续更新中...

**Comment by weidongkl** [comment]
想看这些大数据ai组件的rpm

**Comment by ZhengZhenyu** [comment]
> 想看这些大数据ai组件的rpm

@weidongkl 你好，感谢关注，各开源社区(项目)的发布形式各有不同，像Hadoop就是需要有专门针对于Arm平台的tar包，目前3.3.0已经有Arm对应的发布：https://hadoop.apache.org/releases.html
有些项目比如Spark由于它的pre-build的tar包直接可以在x86和Arm平台上使用，因此只提供了一种tar包的下载:
http://spark.apache.org/downloads.html
而MariaDB则由社区官方维护了Arm平台的RPM: https://mariadb.org/download/
我们后续会把相应的链接和使用指南也刷新到这里
